test/sharness/t0170-dht.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
+
+test_description="Test dht command"
+
+. lib/test-lib.sh
+
+# start iptb + wait for peering
+NUM_NODES=5
+test_expect_success 'init iptb' '
+  iptb init -n $NUM_NODES --bootstrap=none --port=0 &&
+  startup_cluster $NUM_NODES
+'
+
+test_expect_success 'peer ids' '
+  PEERID_0=$(ipfsi 0 id --format="<id>") &&
```

Comment:
can use `iptb get id 0` here (I know its poorly documented, i'm going to work on that soon)